api/src/db/migrations/versions/2025_05_08_add_is_deleted_col_to_user_saved_tables.py

```diff
@@ -0,0 +1,36 @@
+"""add is deleted col to user saved tables
+
+Revision ID: 5fe2cac38b7f
+Revises: b4aad25f47bc
+Create Date: 2025-05-08 19:51:41.692050
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5fe2cac38b7f"
+down_revision = "b4aad25f47bc"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "user_saved_opportunity",
+        sa.Column("is_deleted", sa.Boolean(), nullable=True),
+        schema="api",
+    )
```

Comment:
This doesn't match what the schema says, running the tests will fail because the migrations aren't fully up to date.

You have this as nullable=True, but the SQLAlchemy model doesn't have a nullable column.

If we make that column non-nullable, you have to do some additional work to make it so that column is first populated for everything